.travis/install-cython-examples.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# Cause the script to exit if a single command fails
+set -e
+
+ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
+
+echo "PYTHON is $PYTHON"
+
+cython_examples="$ROOT_DIR/../examples/cython"
+
+if [[ "$PYTHON" == "2.7" ]]; then
+
+   pushd $cython_examples
+   pip install scipy
+   python setup.py install --user
+   popd
+
+elif [[ "$PYTHON" == "3.5" ]]; then
+   export PATH="$HOME/miniconda/bin:$PATH"
+
+   pushd $cython_examples
+   pip install scipy
+   python setup.py install --user
+   popd
+
+elif [[ "$LINT" == "1" ]]; then
```

Comment:
Let's not install anything in the linting builds

Reply:
Ah yes, you're right.